### ISSUE TITLE
I can't see any value in a recient updates (show all parameters in blank)

### ISSUE THREAD

**Issue by hhlp** [penguin-teal/gnome-openweather#49]
## Description

I can't see any value in a recient updates

## System

Which Linux distro are you using?

Fedora : 40
Arch: x64_64

gnome-shell-46.2-1.fc40.x86_64
gnome-shell-extension-openweatherrefined-136-1.fc40.noarch -> Installed from COPR repo

## Settings

```json
Go to extension settings > "About" > "Copy Settings JSON" and paste here.
```

```
{"app-version":"136","git-version":"Desconocido","gnome-version":"46.2","user-locale":"es","appid":"<KEY>","actual-city":"0","sun-in-panel-first":"true","has-run":"true","days-forecast":"5","locs":"[(0, '<NAME>', 0, '<PLACE>')]","owm-api-translate":"true","expand-forecast":"true","show-sunsetrise-in-panel":"true","last-init-error":"'(2024/05/12) TypeError: settings.set_value is not a function'","weatherapidotcom-key":"<KEY>","simplify-degrees":"true","wind-direction":"true","menu-alignment":"71.0","position-in-panel":"'center'","show-comment-in-panel":"true"}
```

![Captura desde 2024-06-23 13-15-54](https://github.com/penguin-teal/gnome-openweather/assets/2659606/55e468a3-4ef6-48ba-93a7-09d82d0dcd27)

Regards.,

**Comment by gramms** [comment]
Try a different weather provider in Settings. I had this issue myself and after switching from "Adaptive" to "OpenWeather", everything was back to normal. Apparently "Adaptive" defaults to "Visual Crossing" in my case, which is not functional.

**Comment by washran** [comment]
Thought I should show my settings json?

{"app-version":"136","git-version":"136.r1.g8d02397","gnome-version":"46.0","user-locale":"en","unit":"'fahrenheit'","appid":"<KEY>","weather-provider":"'weatherapicom'","has-run":"true","days-forecast":"5","refresh-interval-current":"1200","delay-ext-init":"5","locs":"[(0, '<NAME>', 0, '<PLACE>')]","show-sunsetrise-in-panel":"true","my-loc-prov":"'ipinfoio'","weatherapidotcom-key":"<KEY>","position-index":"1","pressure-unit":"'inHg'","position-in-panel":"'center'","wind-speed-unit":"'mph'","show-comment-in-panel":"true"}

I still cannot save api keys for openweather and weatherapi. Openweather is flaky and unstable. Weatherapi is off +_ 2-3 degrees.

**Comment by penguin-teal** [comment]
> Try a different weather provider in Settings. I had this issue myself and after switching from "Adaptive" to "OpenWeather", everything was back to normal. Apparently "Adaptive" defaults to "Visual Crossing" in my case, which is not functional.

It turns out Visual Crossing blocks if API usage goes over, so it is permanently blocked right now unless I appeal which I may or may not do if I think I've found a viable solution (which is unlikely). For the next update it has been removed from the "Adaptive" pool. The reason it would've kept choosing that is most likely because you have high forecast days which the other weather providers do not support.
